assets/dev/js/editor/utils/helpers.js

```diff
@@ -430,6 +430,81 @@ module.exports = {
 		} );
 	},
 
+	/**
+	 * Convert Condition to Conditions
+	 *
+	 * "Condition" is the simple form of Elementor's control conditioning system, which allows to create one or more
+	 * conditions with an "AND" relationship between them.
+	 *
+	 * "Conditions" is the advanced system for conditioning controls, which allows combining AND and/or OR conditions,
+	 * performing checks for larger/smaller than, checking if keys/values contain other keys/values, and more.
+	 *
+	 * This method receives a simple Condition (name and value) and converts it into the advanced Conditions format.
+	 * format.
+	 *
+	 * @since 3.7.0
+	 *
+	 * @param conditionName
+	 * @param conditionValue
+	 * @param controlModel - The control being tested.
+	 * @param values - The containing widget's array of control values.
+	 * @param controls - The containing widget's array of control models.
+	 * @returns {{name, value: ({length}|*), operator: (string)}}
+	 */
+	convertConditionToConditions: function( conditionName, conditionValue, controlModel, values, controls ) {
```

Comment:
Can you write a jest unit test that calls this function? the test will take an old config and convert it to the new config and then you can test some edge cases and stuff. (In case we will have a bug, we can just add another bug case and be sure it does not break the previous cases)